contracts/src/v0.8/vrf/dev/ArbitrumL1Fees.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {ArbGasInfo} from "../../vendor/@arbitrum/nitro-contracts/src/precompiles/ArbGasInfo.sol";
+
+/// @dev An abstract contract that provides Arbitrum specific L1 fee calculations.
+// solhint-disable-next-line contract-name-camelcase
+abstract contract ArbitrumL1Fees {
```

Comment:
Any reason why we consider making ArbitrumL1Fees abstraction as base contracts that can be inherited from, instead of a contract library like ChainSpecificUtil, that can be included in VRFCoordinatorV2_5_Arbitrum contract.

Reply:
I don't think there's a good reason for this. Probably just to keep it consistent with `OptimismL1Fees`

Reply:
Yes, to keep it consistent and to avoid using the utility library. With a utility library, you usually end up using a function that covers different chains and that affects the contract size and execution cost (because the utility function always has to check which chain you're on and then apply the correct logic). By keeping all chain-specific logic in an abstract chain-specific contract, you have a semantic guarantee that the logic inside this chain-specific contract is only meant for this particular chain.

Reply:
Sure